molr-mole-core/src/test/java/io/molr/mole/core/tree/ConcurrentStrandExecutorStepOverParallelTest.java

```diff
@@ -37,27 +39,35 @@
     protected RunnableLeafsMission mission() {
         return new RunnableLeafsMissionSupport() {
             {
-                sequential("step-over", root -> {
-                    parallel = root.parallel("parallel", b -> {
-                        b.sequential("sequential branch A", bA -> {
-                            parallelA1 = bA.run("A.1", () -> {
+                root("step-over").sequential().as(root -> {
+                    root.branch("parallel").parallel().as(b -> {
+                        parallel = latestBlock();
+
+                        b.branch("sequential branch A").sequential().as((Consumer<Branch>) bA -> {
```

Comment:
Why are these casts to `Consumer<Branch>` necessary? They are quite ugly - any way to avoid them?

Reply:
Probably not ... Might have been intellij that tricked me. I will check.  ( think its an old test, where I inlined methods)

Reply:
Fixed